scene/gui/color_mode.h

```diff
@@ -106,20 +112,21 @@ class ColorModeRGB : public ColorMode {
 class ColorModeRAW : public ColorMode {
 public:
 	String labels[3] = { "R", "G", "B" };
-	float slider_max[4] = { 100, 100, 100, 1 };
+	float slider_max[4] = { 1, 1, 1, 1 };
+	float spinbox_max[4] = { 10, 10, 10, 255 };
```

Comment:
Note: The 4th value is never used because we don't apply this to the alpha `SpinBox`. Maybe i can reduce the array size to `3`.